src/Process/Runner/MutationTestingRunner.php

```diff
@@ -100,6 +103,17 @@ public function run(iterable $mutations, string $testFrameworkExtraOptions): voi
 
                 return false;
             })
+            ->filter(function (Mutant $mutant) {
```

Comment:
any reason why this should be done in another `filter()` call instead of the previous one?

Reply:
Yes, this distinction points at a refactoring angle. E.g. you can factor out this filter into another filter-adding class or method to test it separately.